utils_test.py

```diff
@@ -733,15 +734,15 @@ def test_enumerated_partition(self) -> None:
     def test_convert_png_data_url_to_binary(self) -> None:
         image_data_url = '%s%s' % (
             utils.PNG_DATA_URL_PREFIX,
-            python_utils.url_quote(base64.b64encode(b'test123'))) # type: ignore[no-untyped-call]
+            urllib.quote(base64.b64encode(b'test123'))) # type: ignore[no-untyped-call]
```

Comment:
Can probably be moved to the previous line.

Reply:
@vojtechjelinek  Done